windows/check_quiesce_snapshot/check_quiescing_type.yml

```diff
@@ -0,0 +1,91 @@
+# Copyright 2024 VMware, Inc.
+# SPDX-License-Identifier: BSD-2-Clause
+---
+- name: "Initialize not support application consistent quiescing type"
+  ansible.builtin.set_fact:
+    winsrv_app_quiesce_support: false
+
+# Get *vss_manifests*.zip files in VM folder
+- name: "Get all VM files in VM folder"
+  include_tasks: ../../common/vm_get_config.yml
+  vars:
+    property_list:
+      - 'layoutEx.file'
+- name: "Failed to get VM files in VM folder"
+  ansible.builtin.assert:
+    that:
+      - vm_config.layoutEx.file is defined
+      - vm_config.layoutEx.file | length > 0
+    fail_msg: "Can not get VM files info '{{ vm_config.layoutEx.file | default('') }}' after taking snapshot."
```

Comment:
if the assert failed, it means vm_config.layoutEx.file is undefined or no content(length==0). There's always no content can be printed when failure. So '{{ vm_config.layoutEx.file | default('') }}' can be removed from the fail_msg.

Reply:
Yeah, good catch, thanks.